### PR TITLE
add missing await in githubservice

### LINKED ISSUE #354: Bug: comments.filter is not a function
image

Missed an `await`...

### PR DESCRIPTION
**Pull request by prayerslayer**
Fixes #354

### CODE REVIEW

**Comment by taojang** [comment]
:+1:

**Comment by harti2006** [comment]
:+1:

**Comment by prayerslayer** [comment]
👍

File: server/handler/HookHandler.js [hunk 1]
```diff
@@ -48,12 +48,13 @@ class HookHandler {
...
         const zapprFileContent = await this.githubService.readZapprFile(owner.login, name, repo.checks[0].token)
         const zapprfile = new ZapprConfiguration(zapprFileContent)
-        config = zapprfile.isValid() ? zapprfile.getConfiguration() : config
+        config = zapprfile.getConfiguration()
       }
 
       if (Specification.isTriggeredBy(event)) {
-        const token = await getToken(repo, Specification.TYPE)
-        await this.specification.execute(config, payload, token)
+        getToken(repo, Specification.TYPE).then(token =>
```
Comment (gagoman) [inline server/handler/HookHandler.js]: What is a difference?

Comment (prayerslayer) [inline server/handler/HookHandler.js]: Semantically almost nothing and I admit I forgot to revert this. Difference is we're not blocking until specification check is executed before we report back to Github.

**Comment by taojang** [comment]
:+1:

**Pull request merged**